modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/impl/ItMetaStorageServicePersistenceTest.java

```diff
@@ -76,7 +76,8 @@ public void beforeFollowerStop(RaftGroupService service, RaftServer server) thro
 
         var clusterTime = new ClusterTimeImpl(followerNode.name(), new IgniteSpinBusyLock(), new HybridClockImpl());
 
-        metaStorage = new MetaStorageServiceImpl(followerNode.name(), service, new IgniteSpinBusyLock(), clusterTime);
+        metaStorage = new MetaStorageServiceImpl(followerNode.name(), service, new IgniteSpinBusyLock(), clusterTime,
```

Comment:
I'd rather put one param at a line here. It's 4 of them already.

Reply:
done